### ISSUE TITLE
Open Access model

### ISSUE THREAD

**Issue by filippo82** [softwareunderground/subsurface-journal#3]
I have the feeling that Diamond Open Access is what the community wants. However, Diamond OA means that any required funding needs to come from another source.

Has anyone any comments against/about Diamond OA?

**Comment by joferkington** [comment]
For those (like me) who aren't familiar with the terminology of green vs gold vs diamond/etc: 

- https://academia.stackexchange.com/questions/66723/what-is-diamond-open-access
- https://oalibrary.org/diamond-open-access/

**Comment by filippo82** [comment]
@joferkington, these are other good links too:

- https://www.jasonmkelly.com/jason-m-kelly/2013/01/27/green-gold-and-diamond-a-short-primer-on-open-access/
- https://www.ua-foundation.org/diamond-open-access/
- http://legacy.earlham.edu/~peters/fos/overview.htm

**Comment by Japhiolite** [comment]
I also really like the visual representation the Volcanica guys show in the [Introducing Volcanica](https://www.jvolcanica.org/ojs/index.php/volcanica/article/view/16/19) article.  
![image](https://user-images.githubusercontent.com/13364692/54036798-cbc73d00-41bc-11e9-9cd2-6da68ee7b272.png)

**Comment by leouieda** [comment]
I agree that Diamond OA is the goal. I think it's achievable since we're not trying to make a profit. JOSS estimates that it costs them about $3.50 per aticle (https://joss.theoj.org/about#costs). Volcanica doesn't post figures but they have a nice breakdown of the things that cost money in https://www.jvolcanica.org/ojs/index.php/volcanica/policies.

I would be interested in trying to support this through grants and NumFocus or some other foundations might help with that. If we can attract industry attention, we might be able to get some money for supporting the journal from there as well. @kwinkunks has a lot of pull and experience in that sector that we could harness :)

**Comment by JesperDramsch** [comment]
> I agree that Diamond OA is the goal. 

Agreed.

Would it be in the Diamond OA spirit to give the possibility to donate the cost of publishing? Entirely voluntary, but asking usually works wonders in recouping some of the costs.

**Comment by leouieda** [comment]
> Entirely voluntary, but asking usually works wonders in recouping some of the costs.

That's a good idea. Maybe something like the swung voluntary subscription that Matt setup. The trouble would be receiving this money. We'd need a fiscal sponsor like NumFocus.

**Comment by filippo82** [comment]
I think that offering the voluntarily contribute paying a flat fee for the publishing is a good idea. 

I see that the [system used by Volcanica](https://pkpservices.sfu.ca/content/plans-pricing) has a price but it is not prohibitive.

**Comment by JesperDramsch** [comment]
@filippo82 that's only for the hosted version

_Sent from my OnePlus 6 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by filippo82** [comment]
@JesperDramsch good point.

**Comment by filippo82** [comment]
[Open Access 2020 Initiative](https://oa2020.org/be-informed/)
